save-demo/src/main/kotlin/com/saveourtool/save/demo/entity/GithubRepo.kt

```diff
@@ -15,4 +15,9 @@ class GithubRepo(
     var organizationName: String,
     @Column(name = "project_name")
     var toolName: String,
```

Comment:
As I see, for now we have at least **2** different names for single entity, it's a `toolName` and `projectName`. While in terms of Github it's called in third way - `repository`

May be we can open an issue and unify this terms one time?

That's why I asked in other conversation about relation to the `github`, because `tool` associated to me with some executable and not `repository`

Reply:
https://github.com/saveourtool/save-cloud/issues/1645